can/io/logger.py

```diff
@@ -275,18 +277,19 @@ def do_rollover(self) -> None:
         """Perform rollover."""
 
 
-class SizedRotatingLogger(BaseRotatingLogger):
-    """Log CAN messages to a sequence of files with a given maximum size.
+class RotatingLogger(BaseRotatingLogger):
+    """Log CAN messages to a sequence of files with a given maximum size,
+    a specified amount of time or a combination of both. When both max size
+    (bytes) and rollover time (seconds) are provided, the rollover trigger
+    is caused by the first to occur.
 
-    The logger creates a log file with the given `base_filename`. When the
-    size threshold is reached the current log file is closed and renamed
+    The logger creates a log file with the given `base_filename`. When a size
+    or time threshold is reached the current log file is closed and renamed
     by adding a timestamp and the rollover count. A new log file is then
     created and written to.
 
-    This behavior can be customized by setting the
-    :attr:`~can.io.BaseRotatingLogger.namer` and
-    :attr:`~can.io.BaseRotatingLogger.rotator`
-    attribute.
+    This behavior can be customized by setting the :attr:`namer` and
+    :attr:`rotator` attribute.
 
     Example::
```

Comment:
You also have to adjust the example.

Reply:
I've updated the example now, thank you.

Reply:
@felixdivo Please note that the attributes references are restored to the `BaseRotatingLogger` in https://github.com/hardbyte/python-can/pull/1365/commits/2862573c09a2c7b2271169151642af9dcdfb6c27 due to a failure in the `docs` workflow build.